reporter.go

```diff
@@ -74,12 +75,22 @@ func (r *Client) Report() error {
 
 	ticker := time.NewTicker(time.Duration(r.sleep) * time.Second)
 	done := false
+	var memstats runtime.MemStats
 	for !done {
 		select {
 		case key := <-r.flows.Expire:
 			r.flows.Delete(key)
 			log.Infof("Flow expired: [%s]", key)
 		case <-ticker.C:
+			flush := false
+			now := time.Now().Unix()
+
+			runtime.ReadMemStats(&memstats)
+			pct := float64(memstats.Alloc) / float64(memstats.Sys)
```

Comment:
qq:`memstats.Sys` reflects the `bytes obtained from system` by the runtime, is it what you want to know, or would you rather use the total memory of the system?